BUILD.md

```diff
@@ -0,0 +1,203 @@
+# Building the Arkouda Server
+
+## Table of Contents
+1. [Getting Started](#start)
+2. [Environment Variables](#env-vars)
+3. [Dependency Configuration](#dep-config)
+   1. [Using Environment Installed Dependencies *(Recommended)*](#env_deps)
+   2. [Installing Dependencies](#install-deps)
+   3. [Distributable Package](#build-distrib)
+4. [Building the Server](#build-server)
+5. [Building Documentation](#build-ak-doc)
+6. [Modular Builds](#build-ak-mod)
+
+<a id="start"></a>
+## Getting Started <sup><sup><sub><a href="#toc">toc</a></sub></sup></sup>
+ 
+If you have not installed the Arkouda Client and prerequisites, please follow the directions in [INSTALL.md](INSTALL.md) before proceeding with this build.
+
+Download, clone, or fork the [arkouda repo](https://github.com/Bears-R-Us/arkouda). Further instructions assume that the current directory is the top-level directory of the repo.
+
+<a id="env-vars"></a>
+## Environment Variables <sup><sup><sub><a href="#toc">toc</a></sub></sup></sup>
+In order to build the server executable, some environment variables need to be configured. For a full list, please refer to [ENVIRONMENT.md](ENVIRONMENT.md).
+
+<a id="dep-config"></a>
+## Dependency Configuration <sup><sup><sub><a href="#toc">toc</a></sub></sup></sup>
+
+<a id="env-deps"></a>
+### Using Environment Installed Dependencies *(Recommended)* <sup><sup><sub><a href="#toc">toc</a></sub></sup></sup>
+When utilizing a package manager, such as `Anaconda`, to install dependencies (see [INSTALL.md](INSTALL.md)) you will need to provide the path to the location of your installed pacakges. This is achieved by adding the path to your package install location to `Makefile.paths` (Example Below). It is important to note that in most cases you will only provide a single path for your environment. However, if you have manually installed dependencies (such as ZeroMQ or HDF5), you will need to provide each install location.
```

Comment:
typo in `pacakges`. also i really like the rewording

Reply:
yep, fix spelling.

Reply:
Fixed, @pierce314159 good catch thanks.